integration_tests/integration/bookedtoday/arrivals/checkAnswers.spec.ts

```diff
@@ -23,50 +32,92 @@ context('Check Answers', () => {
     sexPage.continue().click()
 
     const imprisonmentStatusPage = ImprisonmentStatusPage.goTo(expectedArrival.id)
-    imprisonmentStatusPage.imprisonmentStatusRadioButton('on-remand').click()
     imprisonmentStatusPage.continue().click()
+    imprisonmentStatusPage.hasError('Select a reason for imprisonment')
+    imprisonmentStatusPage.imprisonmentStatusRadioButton('determinate-sentence').click()
+    imprisonmentStatusPage.continue().click()
+
+    const movementReasonPage = Page.verifyOnPage(MovementReasonsPage)
+    movementReasonPage.continue().click()
+    movementReasonPage.hasError('Select the type of determinate sentence')
+    movementReasonPage.movementReasonRadioButton('26').click()
+    movementReasonPage.continue().click()
 
-    const checkAnswersPage = CheckAnswersPage.goTo(expectedArrival.id)
+    const checkAnswersPage = Page.verifyOnPage(CheckAnswersPage)
 
     checkAnswersPage.name().should('contain.text', 'Sam Smith')
     checkAnswersPage.dob().should('contain.text', '1 February 1970')
-    checkAnswersPage.prisonNumber().should('contain.text', 'G0014GM')
+    checkAnswersPage.prisonNumber().should('contain.text', 'A1234BC')
     checkAnswersPage.pncNumber().should('contain.text', '01/4567A')
     checkAnswersPage.sex().should('contain.text', 'Female')
-    checkAnswersPage.reason().should('contain.text', 'On remand')
+    checkAnswersPage.reason().should('contain.text', 'Determinate sentence - Extended sentence for public protection')
     cy.task('stubCreateOffenderRecordAndBooking', expectedArrival.id)
     checkAnswersPage.addToRoll().click()
-    Page.verifyOnPage(ConfirmAddedToRollPage)
+
+    const confirmAddedToRollPage = Page.verifyOnPage(ConfirmAddedToRollPage)
+    confirmAddedToRollPage.confirmationBanner().should('contain.html', 'Bob Smith')
+    confirmAddedToRollPage.confirmationBanner().should('contain.html', 'A1234AB')
+    confirmAddedToRollPage.confirmationParagraph().should('contain.html', 'Bob Smith')
+    confirmAddedToRollPage.confirmationParagraph().should('contain.html', 'Moorland (HMP &amp; YOI)')
+    confirmAddedToRollPage.locationParagraph().should('contain.html', 'Reception')
+    confirmAddedToRollPage
+      .viewEstablishmentRoll()
+      .should('contain', 'View establishment roll')
+      .should('have.attr', 'href')
+      .then(href => {
+        expect(href).to.equal('https://digital-dev.prison.service.justice.gov.uk/establishment-roll')
+      })
+    confirmAddedToRollPage
+      .backToDigitalPrisonServices()
+      .should('contain', 'Back to Digital Prison Services')
+      .should('have.attr', 'href')
+      .then(href => {
+        expect(href).to.equal('https://digital-dev.prison.service.justice.gov.uk')
+      })
+    confirmAddedToRollPage.addAnotherToRoll().click()
+    Page.verifyOnPage(ChoosePrisonerPage)
   })
 
-  it('Should not display prison or pnc numbers if none present', () => {
+  it('Should show matched results when found', () => {
```

Comment:
there's a load of different options we could test here - this was a bit of an arbitrary choice but had to change existing test as no PNC and no prison number is not a valid combo